src/docs/MLOS/references/Overview.adoc

```diff
@@ -1,9 +1,13 @@
-*Machine Learning Open Studio (ML-OS)* is an interactive graphical interface that enables developers and data scientists to quickly and easily build, train, and deploy machine learning models at any scale. It provides a rich set of generic machine learning tasks that can be connected together to build either basic or complex machine learning workflows for various use cases such as: fraud detection, text analysis, online offer recommendations, prediction of equipment failures, facial expression analysis, etc.
+*Proactive Machine Learning (PML)* is a complete DSML platform (Data Science and Machine Learning) including a ML Studio, AutoML, Data Science Orchestration and MLOps for the deployment,
+training, execution and scalability of artificial intelligence and machine learning models on
+any type of infrastructure. Created for data scientists and ML engineers, the solution is simple to use and accelerates machine learning algorithms toward production.
 
```

Comment:
.....and accelerate the deployment of machine learning algorithms (models ?) in a production environment.